genai/embed.go

```diff
@@ -64,9 +77,46 @@ func (m *EmbeddingModel) EmbedContentWithTitle(ctx context.Context, title string
 		taskType := pb.TaskType(tt)
 		req.TaskType = &taskType
 	}
-	res, err := m.c.c.EmbedContent(ctx, req)
+	return req
+}
+
+// An EmbeddingBatch holds a collection of embedding requests.
+type EmbeddingBatch struct {
+	tt  TaskType
+	req *pb.BatchEmbedContentsRequest
+}
+
+// NewBatch returns a new, empty EmbeddingBatch.
+// Make multiple calls to [EmbeddingBatch.AddContent] or
+// [EmbeddingBatch.AddContentWithTitle].
+// [EmbeddingBatch.EmbedContents] will then get all the embeddings
```

Comment:
This docstring needs updating, since we no longer have this method on the batch

Reply:
done